src/test/java/teammates/test/cases/browsertests/AdminHomePageUiTest.java

```diff
@@ -86,12 +86,12 @@ private void testCreateInstructorAction() throws Exception {
         BackDoor.deleteCourse(demoCourseId);
         BackDoor.deleteInstructor(demoCourseId, instructor.email);
         homePage.createInstructorByInstructorDetailsSingleLineForm(instructorDetails);
```

Comment:
More on the retry things : http://stackoverflow.com/questions/8295100/how-to-re-run-failed-junit-tests-immediately

Can catch the AssertionError throw by Junit ?